test/connectivity/modules/NetworkController/docker/linux/arm32v7/Dockerfile

```diff
@@ -1,27 +1,21 @@
 # syntax=docker/dockerfile:1.4
 
-FROM mcr.microsoft.com/dotnet/aspnet:6.0-focal-arm32v7
-
-RUN apt-get update && apt-get install -y libcap2-bin libsnappy1v5 && \
-    ln -s /lib/arm-linux-gnueabihf/libdl.so.2 /usr/lib/arm-linux-gnueabihf/libdl.so && \
-    ln -s /lib/arm-linux-gnueabihf/libc.so.6 /usr/lib/arm-linux-gnueabihf/libc.so && \
-    rm -rf /var/lib/apt/lists/*
-
-COPY $EXE_DIR/librocksdb/librocksdb.so.armhf /usr/lib/librocksdb.so
-
-RUN apt-get update && apt-get install -y --no-install-recommends iproute2 net-tools
-
-# Add an unprivileged user account for running the module
-RUN useradd -ms /bin/bash moduleuser
-USER moduleuser
+FROM mcr.microsoft.com/dotnet/aspnet:6.0-alpine
 
 ARG EXE_DIR=.
 
+RUN apk update && \
+    apk add --no-cache iproute2
```

Comment:
Aren't we missing net-tools and copying rocksdb here in this dockerfile?

Reply:
Hmm...yes it looks like I dropped those steps. Let me add them back, and I can run the connectivity pipeline to ensure I migrated them properly.

Reply:
@and-rewsmith It looks like the NetworkController Dockerfile for amd64 _never_ had rocksdb, and the arm32/64 didn't have it either until a few months ago, when David changed the way arm32/64 modules are built. Are you sure NetworkController needs rocksdb and net-tools?

All I did was make the arm32/64 Dockerfiles consistent with the amd64 Dockerfile.

Reply:
Ok, I'll make sure the connectivity tests pass with the NetworkController image changes as they currently stand.